### PR TITLE
swift docs update - logging + sdk improvements

### PR DESCRIPTION
**Pull request by kp-cat**
### Related tickets

https://trello.com/c/o69PnArg/168-sdk-loglevel-update-swift
https://trello.com/c/5TtFIjjG/148-sdk-improvements-swift

### Description

Swift Docs update:
- Logging
- Remove `maxWaitTimeForSyncCallsInSeconds`
- Add `maxInitWaitTimeInSeconds`
- Add `flagOverrides`

Python Docs update:
- Logging

### Requirement checklist

- [ ] I have validated my changes on a test/local environment.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

**Review by z4kn4fein — APPROVED** [review]

File: website/docs/sdk-reference/ios.md [hunk 1]
```diff
@@ -150,15 +149,26 @@ You can change the verbosity of the logs by passing a `logLevel` parameter to th
...
+| ---------- | --------------------------------------------------------------------------------------- |
+| `.nolog`   | Turn the ConfigCat logging off.                                                         |
+| `.error`   | Only error level events are logged.                                                     |
+| `.warning` | Errors and Warnings are logged.                                                         |
+| `.info`    | Errors, Warnings and feature flag evaluation is logged.                                 |
+| `.debug`   | All of the above plus debug info is logged. Debug logs can be different for other SDKs. |
 
 > The default level is `.warning`. With `.nolog` you can turn the logging completely off.
 
+Info level logging helps to inspect the feature flag evaluation process:
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/ios.md:165]:
```suggestion
Info level logging helps to inspect the feature flag evaluation process,
Example log entries:
```

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

**Review by sigewuzhere — DISMISSED** [review]

**Review by sigewuzhere — APPROVED** [review]

**Review by sigewuzhere — APPROVED** [review]

**Review by sigewuzhere — APPROVED** [review]

File: website/docs/sdk-reference/ios.md [hunk 2]
```diff
@@ -72,10 +72,9 @@ if(isMyAwesomeFeatureEnabled) {
...
 | ---------------------------------- | ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `sdkKey`                           | string                  | **REQUIRED.** SDK Key to access your feature flags and configurations. Get it from *ConfigCat Dashboard*.                                                                                                                                                                                           |
-| `dataGovernance`                   | DataGovernance          | Optional, defaults to `global`. Describes the location of your feature flag and setting data within the ConfigCat CDN. This parameter needs to be in sync with your Data Governance preferences. [More about Data Governance](advanced/data-governance.md). Available options: `global`, `euOnly`. |
```
Comment (z4kn4fein) [inline website/docs/sdk-reference/ios.md:75]: loglevel is missing from this table

File: website/docs/sdk-reference/ios.md [hunk 3]
```diff
@@ -150,25 +152,55 @@ or check <a href="https://developer.apple.com/videos/play/wwdc2016/721" target="
...
+
+| Level      | Description                                                                             |
+| ---------- | --------------------------------------------------------------------------------------- |
+| `.nolog`   | Turn the ConfigCat logging off.                                                         |
+| `.error`   | Only error level events are logged.                                                     |
+| `.warning` | Errors and Warnings are logged.                                                         |
+| `.info`    | Errors, Warnings and feature flag evaluation is logged.                                 |
+| `.debug`   | All of the above plus debug info is logged. Debug logs can be different for other SDKs. |
 
 > The default level is `.warning`. With `.nolog` you can turn the logging completely off.
```
Comment (z4kn4fein) [inline website/docs/sdk-reference/ios.md:168]:
```suggestion
```

File: website/docs/sdk-reference/ios.md [hunk 4]
```diff
@@ -150,25 +152,55 @@ or check <a href="https://developer.apple.com/videos/play/wwdc2016/721" target="
...
-- `.error`
-- `.nolog`
+
+Available log levels:
+
+| Level      | Description                                                                             |
+| ---------- | --------------------------------------------------------------------------------------- |
+| `.nolog`   | Turn the ConfigCat logging off.                                                         |
+| `.error`   | Only error level events are logged.                                                     |
+| `.warning` | Errors and Warnings are logged.                                                         |
```
Comment (z4kn4fein) [inline website/docs/sdk-reference/ios.md:164]:
```suggestion
| `.warning` | Default. Errors and Warnings are logged.                                                         |
```

**Review by z4kn4fein — CHANGES_REQUESTED** [review]

**Review by z4kn4fein — APPROVED** [review]

**Review by sigewuzhere — APPROVED** [review]

**Pull request merged**